src/EnergyPlus/PlantHeatExchangerFluidToFluid.cc

```diff
@@ -1998,4 +2001,101 @@ void HeatExchangerStruct::oneTimeInit(EnergyPlusData &state)
     }
 }
 
+void HeatExchangerStruct::updateCompFlowData(EnergyPlusData &state)
```

Comment:
The fluid to fluid HX logic copies the coil data from the plant loop on the supply side of the HX (downstream side, i.e., to a secondary loop) to the plant loop on the demand side of the HX (upstream side). This moves the comp flow rates (load surrogate) to the upstream plant.